package-examples/helloworld/README.md

```diff
@@ -0,0 +1,53 @@
+# helloworld
+
+Sample helloworld package.  Showcases what a bare bones package is which doesn't do anything beyond declaring the current directory as a `kpt` package. 
```

Comment:
Nit: Better to have a new line after 80 chars approx. We need not add it as a lint check in future.